release notes/v0.9.0.md

```diff
@@ -0,0 +1,111 @@
+
+## New features
+
+### `browserContext.addCookies` [xk6-browser#760](https://github.com/grafana/xk6-browser/pull/760)
+
+Websites require cookies to help store session details which will be used by the backend to retrieve the state. For example, when a user logins into a website then the backend will store login session details in a cookie, which can be used in future actions performed on the website. For the given example, to speed up the test run, instead of having to first go through the login flow, it would be quicker to set a known valid cookie so that the user can easily test a different feature of the website. With this new feature cookies can be added to a `browserContext` and all new `page`s created from this `browserContext` will have the cookie assigned to them. Thanks @zucchinho for implementing the feature!
+
+```go
+  const context = browser.newContext()
+  context.addCookies([{name: 'myCookie', value: 'hello world', url: 'https://test.k6.io'}])
+  const page = context.newPage()
+```
+
+### `browserType.Connect` [xk6-browser#800](https://github.com/grafana/xk6-browser/pull/800)
+
+There are cases where the user may want to connect to a remote browser instance where they have more control over the browser lifecycle, such as when working in a resource bound environment. This feature enables users to connect to a manually started Chrome/Chromium browser instance. It's a simple case of replacing `browser.launch` with `browser.connect` and supplying the CDP url as the first argument. Not all `launch` options will work with `connect` since the browser instance should already have started prior to working with `connect`. Since we assume that the user had decided to take ownership of starting the browser, we have made `browser.close` a NOOP when working with `browser.connect`, so the user will need to close the browser themselves.
+
+```go
+  const browser = chromium.connect('ws://127.0.0.1:1234/devtools/browser/e3bb7e53-ad0f-46f3-ae89-a8416868f4ce')
+  const page = browser.newPage();
+```
+
+### Web Vitals are now natively supported by the k6-browser module [xk6-browser#836](https://github.com/grafana/xk6-browser/pull/836)
+
+Web vitals are the defacto way for developers to measure their frontend performance using the core metrics:
+
+- Largest contentful paint ([LCP](https://web.dev/lcp/))
+- First input delay ([FID](https://web.dev/fid/))
+- Cumulative layout shift ([CLS](https://web.dev/cls/))
+
+These measurements are now calculated for all tests without any additional work from your side. Simply run your test as you have been doing and you will be presented with the metrics in the output as you have used other measurements. This is the output after running the [examples/fillform.js](https://github.com/grafana/xk6-browser/blob/main/examples/fillform.js) script:
```

Comment:
Just adding a comment that I don't quite understand the part here that says: "as you have used other measurements." But it might be ok :+1:

Reply:
Yeah, that doesn't make sense. I've removed that part. Resolved in d5e14dab29cea3af9bf07217f6543261d1c2f4b9